cmd/snapshot-create.go

```diff
@@ -0,0 +1,161 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/Layr-Labs/sidecar/internal/logger"
+	"github.com/spf13/cobra"
+)
+
+var (
+	snapshotCreateToPath string
```

Comment:
Rename to `output` that specifically takes a file path and also consider defaulting to writing to stdout if `--output` is omitted.